sktime/forecasting/base/adapters/_generalised_statsforecast.py

```diff
@@ -254,3 +271,197 @@ def _predict_interval(self, fh, X, coverage):
         final_interval_predictions = pandas.concat(interval_predictions, copy=False)
 
         return final_interval_predictions
+
+
+class StatsForecastBackAdapter(_TS):
```

Comment:
Related to above comment, is it necessary to inherit from `_TS`?

First of all it's private I think (from starting _), plus you're only using `new` method from it as far as I can see, and I did not understand why is that necessary. Can you please explain?

Reply:
I just assumed it was necessary to do this, I think you're right that it's not necessary